README.md

```diff
@@ -138,6 +138,47 @@ Will result in:
 </div>
 ```
 
+## Custom widgets
+
+You can provide your own custom widgets to a uiSchema for the following json data types:
+
+- `string`
+- `number`
+- `integer`
+- `boolean`
+- `date-time`
+
+```js
+const schema = {
+  type: "string"
+};
+
+const uiSchema = {
+  widget: (props) => {
```

Comment:
Well, that's the way you set a widget in uiSchema. Thought it was clear enough, but oh well :)